tests/common/models.py

```diff
@@ -4,7 +4,7 @@
 """Contains commonly used models that are shared across the test suite."""
 import copy
 from functools import partial
-from typing import Any, Dict, Tuple, Union
+from typing import Any, Dict, Optional, Tuple, Union
```

Comment:
Do you want to move these files to another PR -- they seem related to another task?

Reply:
This is because the test is failing and blocking this PR :(